tox.ini

```diff
@@ -20,7 +14,7 @@ deps =
     h5py
     pint
     astropy
-    coverage<5.0
```

Comment:
Not sure exactly what was having problems with `coverage`, but having `coverage<5.0` was causing `matplotlib` and `numpy` to get into fights and have tox try to step backwards through releases until they failed out.

Reply:
Actually, this is still happening on: https://github.com/yt-project/unyt/actions/runs/1105527314

Can someone with admin rights cancel that workflow?

Reply:
done

Reply:
Can you also get this one: https://github.com/yt-project/unyt/actions/runs/1105527310

Reply:
you got it